cast/js/rhino/src/main/java/com/ibm/wala/cast/js/translator/RhinoToAstTranslator.java

```diff
@@ -1596,6 +1600,40 @@ public CAstNode visitSymbol(Symbol node, WalkContext arg) {
       return null;
     }
 
+    @Override
+    public CAstNode visitTemplateCharacters(TemplateCharacters node, WalkContext arg) {
+      return Ast.makeConstant(node.getValue());
+    }
+
+    @Override
+    public CAstNode visitTemplateLiteral(TemplateLiteral node, WalkContext arg) {
```

Comment:
I can't really understand the logic here; I think we need some comments to explain.  Is the code in this method handling just the string concatenation?